var/spack/repos/builtin/packages/libgcrypt/package.py

```diff
@@ -32,7 +32,7 @@ class Libgcrypt(AutotoolsPackage):
        key algorithms, large integer functions, random numbers and a lot
        of supporting functions. """
     homepage = "http://www.gnu.org/software/libgcrypt/"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
```

Comment:
A better question would be, why is this change necessary in the first place? Was there something wrong with the old download URL?